customersupport/controllers/ticket.py

```diff
@@ -1,32 +1,68 @@
 from customersupport import app
-from customersupport.wrappers import sales
-from customersupport.models import Ticket, CallLog
+from customersupport.wrappers import sales, hr
+from customersupport.models import Ticket, CallLog, TicketType, TicketStatus
 from customersupport.database import db_session
-from customersupport.util import get_param_from_request_if_not_empty
+from customersupport.util import get_param_from_request_if_not_empty, get_post_data
 
-from flask import Flask, request, render_template
+from flask import Flask, request, render_template, abort
 from flask import request
 from flask import render_template
 from flask import jsonify
 from flask import json
 
-@app.route('/sales/refund/real')
-def refund_order(mock=False):
-    order = sales.initiate_refund(
-        replace=True,
-        order_id=1,
-        serial_numbers=[
-            123,
-            456,
-            789
-        ],
-        mock=mock
+@app.route('/api/refund', methods=['POST'])
+def refund_order():
+
+    use_mock = get_post_data("use_mock") is not None
+    replace = get_post_data("replace") == "true"
+    ticket_type = TicketType.REFUND
+    if replace:
+        ticket_type = TicketType.REPLACE
+    status = get_post_data("status")
+    current_status = TicketStatus.CLOSED
+    if status == "open":
+        current_status = TicketStatus.OPEN
+    elif status == "pend":
+        current_status = TicketStatus.PENDING
+    elif status == "pass" or status == "fail":
```

Comment:
I don't remember if we talked about this. Would it be a problem to add the "Closed - Successful" and "Closed - Failure" states on the ticket? That way, it would show in the "List Tickets" view.